template/provision/group_vars/all.erb

```diff
@@ -5,10 +5,12 @@ use_sudo: true
 
 # ruby-common
 # pull the checksum/url from https://www.ruby-lang.org/en/downloads/
-ruby_version: ruby-2.2.5
-ruby_checksum: 30c4b31697a4ca4ea0c8db8ad30cf45e6690a0f09687e5d483c933c03ca335e3
-ruby_download_location: 'http://cache.ruby-lang.org/pub/ruby/ruby-2.2.5.tar.gz'
+ruby_version: ruby-2.7.1
+ruby_checksum: d418483bdd0000576c1370571121a6eb24582116db0b7bb2005e90e250eae418
+ruby_download_location: https://cache.ruby-lang.org/pub/ruby/2.7/ruby-2.7.1.tar.gz
+bundler_version: 2.1.4
 
-# passenger
-passenger_webserver: apache
-passenger_pkgs_state: present
+# Other stuff
+letsencrypt_email: 
+nodejs_version: 13.x
+ssl_enabled: true
```

Comment:
Oh nice can you add `postgresql_version: 11.7` here too? right now we default to 9.4 and its pretty bad